test/app-tap/datetime.test.lua

```diff
@@ -554,6 +554,85 @@ test:test("Check parsing of dates with invalid attributes", function(test)
     end
 end)
 
+test:test("Parsing of timezone abbrevs", function(test)
+    test:plan(156)
+    local zone_abbrevs = {
```

Comment:
I would place one abbrev to a single line for better readability and for readable diffs in the future.

Reply:
I'd rather prefer to not - that would make this particular array declaration look .. hmm, weaker. Less compact. (This is for my personal tastes, yes)

Reply:
Okay.